sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterMetricExtensions.cs

```diff
@@ -9,7 +9,7 @@ namespace Azure.Monitor.OpenTelemetry.Exporter
     /// <summary>
     /// Extension methods to simplify registering of Azure Monitor Metrics Exporter.
     /// </summary>
-    internal static class AzureMonitorExporterMetricExtensions
+    public static class AzureMonitorExporterMetricExtensions
```

Comment:
we don't allow users to pick the interval, right?

Reply:
Correct. We don't have an option at this moment.